src/main/java/nl/dtls/fairdatapoint/api/config/ApplicationInitializer.java

```diff
@@ -27,17 +27,18 @@
 
 /**
  * Application config (Similar to web.xml)
- * 
+ *
  * @author Rajaram Kaliyaperumal <rr.kaliyaperumal@gmail.com>
  * @author Kees Burger <kees.burger@dtls.nl>
  * @since 2015-11-19
  * @version 0.1
  */
 public class ApplicationInitializer extends
         AbstractAnnotationConfigDispatcherServletInitializer {
+
     @Override
     protected Class<?>[] getRootConfigClasses() {
-        return new Class[]{ RootConfig.class, RestApiContext.class };
+        return new Class[]{RootConfig.class, RestApiContext.class};

```

Comment:
I'd argue the old style is more pleasant for the eyes. Which style format did you use?